### DISCUSSION TITLE
Bonus Town Blocks

### DISCUSSION THREAD

**Discussion by dragontekgaming** [TownyAdvanced/Towny#5796] [Q&A] [answered]
I would like to give players bonus town blocks based on various things like Voting for the server, Playtime, or in a special crate for mining. Unfortunately they get a big blurb about the Minecraft EULA, Towny License and about purchasing the blocks. Is there no way around this? I can understand wanting to make sure people aren't selling things, and I wouldn't dream of doing so, but this is an aggrivation.

**Comment by LlmDl** [comment, marked as answer]
Imagine the aggravation felt when people are selling townblocks though.

**Reply by dragontekgaming** [reply]
I can certainly understand the aggravation. I just wish it could be reduced somehow. For example:

[TOWNY] If you have received these town blocks in exchange for money, please report it here: [LINK], where the link leads to either here, or maybe to http://aka.ms/mce-reportserver. 

I'm just trying to think of a way to make the use not fill half the chat each time someone uses an item that gives them the blocks, or if the command is run to give them the blocks.
